index.html

```diff
@@ -19,5 +19,37 @@ <h1>Experience the rush today</h1>
                 </div>
             </div>
         </section>
+        <section class = "sect_content">
+            <div class = "container">
+                <div class="feature">

```

Comment:
Using `dl/dt/dd` element might be more semantically correct.

https://www.w3schools.com/tags/tag_dd.asp
https://css-tricks.com/utilizing-the-underused-but-semantically-awesome-definition-list/

Read about semantic html.

Reply:
@probil, can you add a separate issue for this?